torchgen/gen_lazy_tensor.py

```diff
@@ -347,20 +351,22 @@ def sort_native_function(f: Union[NativeFunctionsGroup, NativeFunction]) -> str:
     grouped_native_functions = sorted(
         grouped_native_functions, key=sort_native_function
     )
+
     parsed_backend_yaml = parse_backend_yaml(
         source_yaml, grouped_native_functions, backend_indices
     )
     backend_key = parsed_backend_yaml.backend_key
     autograd_key = parsed_backend_yaml.autograd_key
     cpp_namespace = parsed_backend_yaml.cpp_namespace
     backend_indices = parsed_backend_yaml.backend_indices
-    full_codegen, non_native = parse_native_functions_keys(
+    full_codegen, non_native, ir_gen = parse_native_functions_keys(
```

Comment:
Some comments on why we have these 3 flavors of op names would be great

Reply:
added comments